### PR TITLE
OBPIH-3758 Fix order adjustment as invoice items

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: src/js/components/invoice/AddItemsPage.jsx [hunk 1]
```diff
@@ -120,7 +121,7 @@ const FIELDS = {
         defaultMessage: 'Unit Price',
         flexWidth: '1',
         attributes: {
-          formatValue: value => (value ? (value.toFixed(2)) : value),
+          formatValue: value => (value ? accountingFormat(value.toFixed(2)) : value),
```
Comment (pmuchowski) [inline src/js/components/invoice/AddItemsPage.jsx:124]: I think you can move `toFixed(2)` to this new utils method

File: grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy [hunk 2]
```diff
@@ -81,29 +95,58 @@ class InvoiceItem implements Serializable {
...
+    String getOrderNumber() {
+        return shipmentItem?.orderNumber ?: orderAdjustment?.order?.orderNumber ?: null
+    }
+
+    String getDescription() {
+        return orderAdjustment ? orderAdjustment.description : product?.name
+    }
+
+    Order getOrder() {
+        return orderNumber ? Order.findByOrderNumber(orderNumber) : null
```
[the remark this answers is not in the document]
Comment (awalkowiak) [reply grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy:119]: It was done that way before, I only moved it. Also for invoice items for shipments, you'd need to look into something else, so this looks cleaner.

[the remark this answers is not in the document]
Comment (awalkowiak) [reply grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy:119]: It was done this way before, I just moved it. There is also a shipment case. This way looks cleaner I think

[the remark this answers is not in the document]
Comment (awalkowiak) [reply grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy]: I changed it. Now I am using the getOrderItem.

**Review by DominikaHatala — APPROVED** [review]

**Review by pmuchowski — APPROVED** [review]

**Pull request merged**